src/test/java/teammates/common/datatransfer/questions/FeedbackMsqQuestionDetailsTest.java

```diff
@@ -210,6 +288,116 @@ public void testValidateResponseDetails_choiceNotInValidChoices_shouldTriggerErr
         assertEquals(0, errors.size());
     }
 
+    @Test
+    public void testValidateResponseDetails_otherDisabledButExists_shouldReturnError() {
```

Comment:
```suggestion
    public void testValidateResponsesDetails_otherDisabledButExists_shouldReturnError() {
```

Would be good to make this change to all `testValidateResponseDetails` tests so that the name is consistent with the name of the function being tested, as well as the MCQ tests.

Reply:
Thank you! Renamed methods as you said.

Reply:
> Thank you! Renamed methods as you said.

Now we have `testValidateResponsesDetails` for MCQ tests but `tValidateResponsesDetails` for MSQ tests, which is still inconsistent. If the length of the testcase name is too long, you may shorten the name if necessary, but please still keep the naming consistent across files. I would prefer keeping `testValidateResponsesDetails` but shortening any individual testcases with names that are too long (e.g. `testValidateResponsesDetails_otherAnsNotChosenButOtherFieldNotEmpty_shouldReturnError`).

Reply:
Sorry for taking long break, but I renamed all tValidateResponsesDetails tests, now they all seem consistent.